source/irfs/effective_area/index.rst

```diff
@@ -54,4 +52,34 @@ sources.
 * ``RAD_MAX`` type: float, unit: deg
     * On region radius for point-like observations
 
+Although not a requirement, the recommended ``EXTNAME`` keyword is "EFFECTIVE AREA".
+
+.. _aeff_reco_2d:
+
+Effective Area vs reconstructed energy
+--------------------------------------
+
+Although not required, it is generally useful to store the effective area as a function of the reconstructed energy, 
```

Comment:
> Although not required, it is generally useful

Can you rephrase here?

Both (AEFF, EDISP) and the combined AEFF_RECO are in use for current IACTs and both are useful.
Whether CTA could just pick one and never need the other is unclear to me ... my guess would be no, i.e. both will be used.

* Using AEFF, EDISP is more precise, but it requires enough MC stats to have an EDISP with acceptable noise, and it's slower to use in likelihood fits for science tools.
* Using AEFF_RECO is less precise, requires less MC stats and is faster to use in likelihood fits for science tools.

Maybe it's OK to extend the description of this format to explain this briefly why they both exist and when to use which one as DL3 producer (i.e. roughly the short summary I wrote here)?